### ISSUE TITLE
script exits with error if incorrect permissions in API/refresh token

### ISSUE THREAD

**Issue by fr0gger03** [vmware/python-client-for-vmware-cloud-on-aws#96]
### Describe the bug

If a user has incorrect permissions in their refresh token, today the script simply exits without explaining why.  The user receives a "403" error - we should capture that and explain it is most likely due to incorrect permissions.

### Reproduction steps

```bash
1. Configure a refresh token with Org Member privileges
2. Attempt to run ./pyVMC.py show-csp-groups (requires org owner)
3.
...
```

### Expected behavior

As a user, I would expect to receive a clear error telling me I do not have permissions, rather than an arcane "403
![2022-08-08_09-17-46](https://user-images.githubusercontent.com/5892826/183427132-05c12082-ced0-46d4-90a5-b2e0a1e2b607.png)
"

**Comment by BillRothVMware** [comment]
Tom, I'll reach out about this one.

**Comment by BillRothVMware** [comment]
created  fork for this.

**Comment by BillRothVMware** [comment]
Fixed with Merge #100 .

**Comment by BillRothVMware** [comment]
ok. not fixed. working on this.

**Comment by BillRothVMware** [comment]
OK. Now fixed in #116 .

**Comment by BillRothVMware** [comment]
@fr0gger03 please close.
